searx/botdetection/install.py

```diff
@@ -6,33 +6,38 @@
 import sys
 import flask
 
-from searx import redisdb
-from searx.plugins import logger
+from searx import redisdb, logger
 from searx.botdetection import limiter
 
-name = "Request limiter"
-description = "Limit the number of request"
-default_on = False
-preference_section = 'service'
-
+# the configuration are limiter.toml and "limiter" in settings.yml
+# so, for coherency, the logger is "limiter" even if the module name "searx.botdetection"
 logger = logger.getChild('limiter')
 
 
+_INSTALLED = False
+
+
 def pre_request():
     """See :ref:`flask.Flask.before_request`"""
     return limiter.filter_request(flask.request)
 
 
-def init(app: flask.Flask, settings) -> bool:
+def is_installed():
+    return _INSTALLED
+
+
+def initialize(app: flask.Flask, settings):
+    """Instal the botlimiter aka limiter"""
+    global _INSTALLED  # pylint: disable=global-statement
     if not settings['server']['limiter'] and not settings['server']['public_instance']:
-        return False
+        return
```

Comment:
Please Markus, is it possible for you to deal with choice that are not yours, like others do?
Yesterday, you said you want to do your PR, then switch back to my PR when I suggest to create myself a package for botdetection. The underlying question was not what to do, but the fact you want to ownership of the code. This is exhausting, and what remains in mind my mind is the fact that SearXNG is a playground and want to do whatever you want.